distributed/client.py

```diff
@@ -3156,7 +3156,17 @@ def _graph_to_futures(
             from distributed.protocol.serialize import ToPickle
 
             header, frames = serialize(ToPickle(dsk), on_error="raise")
-            nbytes = len(header) + sum(map(len, frames))
+
+            def safelen(o):
+                try:
+                    return len(o)
+                except TypeError:
+                    # https://github.com/python/cpython/pull/18463/files
+                    if isinstance(o, memoryview) and o.ndim == 0:
+                        return 1
+                    raise
```

Comment:
Coverage says it's untested (but it's been very unreliable lately); could you point to the test?

Reply:
distributed/shuffle/tests/test_rechunk.py::test_rechunk_0d

Reply:
The original code was wrong to begin with. It should have used memoryview.nbytes otherwise it would give wrong results for numpy or parquet objects with multi-byte dtypes.

I've reverted the change here and move it to #8268, which is now a blocker.